lib/migrations/migrate/stub/knexfile-ts.stub

```diff
@@ -1,7 +1,8 @@
-// Update with your config settings.
+import type { Knex } from "knex";
 
-module.exports = {
+// Update with your config settings.
 
+const config: { [key: string]: Knex.Config } = {
```

Comment:
why [key: string]?

Reply:
It's an object with strings as the property key names and config objects as the property values. Is that not what knex expects?

Reply:
isn't just Knex.Config sufficient?

Reply:
oh, wait, this is for knexfile. Right